test/fixtures/minimal/main.tf

```diff
@@ -15,7 +15,57 @@
  */
 
 module "example" {
-  source     = "../../../examples/minimal"
```

Comment:
Why are we changing this?

Reply:
The example is pinned to provider version 3.53 but we want the test to run on 4.x. 

I could either:
- adjust version pin to `">=3.53"` which is the same as not pinning anything.
- keep the pin in the example but duplicate the copy

Reply:
@upodroid ah I see its referencing the module in the registry as its part of a c.g.c doc. We can skip bumping the provider versions on this example. Since its referencing a released module this should be okay.

Reply:
done, pinned it back to v3.62 but with new provider versioning style

Reply:
@upodroid Can you revert the changes to the fixture too?